modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapInsertItem.vue

```diff
@@ -105,6 +107,9 @@ function closeInsertMenuItem() {
   removeSlash();
   emit('set-active-insert-menu', false);
   isInlineComponentActive.value = false;
+  if (contextMenu.value) {
+    contextMenu.value.hide();
+  }
 }
 
```

Comment:
this file changes are fixing this bug:
https://linear.app/apostrophecms/issue/PRO-7392/rich-text-widget-image-insertion-menu-not-working-correctly.

We simply close the context menu when closing the insert menu item to avoid some listeners to still activated.